src/pages/Popup/Popup.jsx

```diff
@@ -99,12 +114,28 @@ const Popup = () => {
     }
   }
 
+  // function to handle when the play ended
+  function finishedPlaying(message) {
+    //handle when play ended
+    if (buttonPlayActive) playHandler();
+    //parse the finishedPlaying message on textarea
+    setTextContent(message);
```

Comment:
Why do we need to add the text into the text area after playing? I would remove as we want to keep the textarea with the instructions to play again. This should only be changed after recording.